### ISSUE TITLE
最终合成的步骤可能占用内存过大的问题？

### ISSUE THREAD

**Issue by T-99w** [bytedance/LatentSync#50]
在最后合成视频的时候发现可能存在爆内存的风险，16s的视频合成也比较久内存干到快42G左右,CPU资源也直接占用拉满（这属于正常么？）

![微信图片_2025-01-08_230945_977](https://github.com/user-attachments/assets/695b1e93-fafd-448a-8f34-10e5a0d9be88)

=======================
机器配置如下：
![屏幕截图 2025-01-08 231047](https://github.com/user-attachments/assets/b560dbef-c883-4952-8e87-0144b8e18531)

**Comment by soheildousti** [comment]
Can you tell me the steps for executing the code from the beginning to training?

**Comment by T-99w** [comment]
> Can you tell me the steps for executing the code from the beginning to training?

Of course, I use the gradio_app.py to upload videos and audio from the web page. After submission, I wait for the completion of all the progress in the terminal. When the last progress is synthesized into a video, the progress will be very slow and occupy a lot of running memory.

**Comment by soheildousti** [comment]
Can you give me a way to communicate? I'm in trouble. Can you help me? Please?

**Comment by T-99w** [comment]
> Can you give me a way to communicate? I'm in trouble. Can you help me? Please?

Which step did you have difficulty performing? If I could help

**Comment by soheildousti** [comment]
> > Can you give me a way to communicate? I'm in trouble. Can you help me? Please?
> 
> Which step did you have difficulty performing? If I could help

I didn't understand the step-by-step process of the project to train my data model. If you could tell me how I should run it,

**Comment by soheildousti** [comment]
Can you please help me?

**Comment by chunyu-li** [comment]
@soheildousti 
I already pushed the code in commit b3111742e01ad88cb83f765ceab849a61ab72cdc, just pull the latest code.

**Comment by soheildousti** [comment]
Can this code get the accuracy and loss? Or does it just make a video?
Can you tell me the pre-processing and training steps, how should I do it?

**Comment by T-99w** [comment]
@soheildousti You can refer to （Data Processing Pipeline） and  （Training SyncNet），The content of this part

**Comment by soheildousti** [comment]
Were you able to TRAIN? I tried on https://github.com/Rudrabha/Wav2Lip/tree/master but the loss = 0.69 could not be reduced. I tried everything. I wanted to know if this would work?
